### PR TITLE
Add guideline to add a new worker node

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Specify your PR reviewers**:

/cc @gyliu513 @jichenjc @xunpan

**Special notes for your reviewer**:

/sig ibmcloud

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,6 +197,10 @@ your IBM Cloud Cluster API Kubernetes cluster.
    rm -rf $HOME/.ssh/id_ibmcloud*
    ```
 
+### Create additional worker nodes
+
+Please refer to [Trouble shooting documentation](docs/add_node.md) for further info.
```
Comment (gyliu513) [inline README.md]: Can you put the document in this file directly? I think adding a node should not belong to troubleshooting section.

Comment (jichenjc) [reply README.md]: a typo..

File: docs/add_node.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+NAME                  PROVIDERID              PHASE
+jichen-master-g2x5d   xxxxxxxx                Running
+jichen-node-s9s2b     xxxxxxxx                Running
+```
+
+In order to add additional node, need create a machine definition,
+refer to `examples/ibmcloud/out/machines.yaml` for more info.
+
+Here's a sample and we can put into `~/temp.yaml`
+```
```
Comment (gyliu513) [inline docs/add_node.md]: add yaml here to highlight it.

File: docs/add_node.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Add additional nodes
+## How to add a new node after cluster creation completed
+after cluster creation with command `clusterctl`, machines can be queried by:
+```
+# kubectl --kubeconfig=kubeconfig  get machines
+NAME                  PROVIDERID              PHASE
+jichen-master-g2x5d   xxxxxxxx                Running
+jichen-node-s9s2b     xxxxxxxx                Running
```
Comment (xunpan) [inline docs/add_node.md]: What about `PROVIDERID` as `ibmcloud:////xxxxxxxx`?

File: docs/add_node.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+In order to add additional node, need create a machine definition,
+refer to `examples/ibmcloud/out/machines.yaml` for more info.
+
+Here's a sample and we can put into `~/temp.yaml`
```
Comment (xunpan) [inline docs/add_node.md]: What about an example file under examples or use here document instead?

Comment (jichenjc) [reply docs/add_node.md]: I prefer to use document first... all other providers seems not provide this in template.

**Pull request merged**